lib/ui/window/window.h

```diff
@@ -27,6 +29,7 @@ class Window final {
   const ViewportMetrics& viewport_metrics() const { return viewport_metrics_; }
 
   void DispatchPointerDataPacket(const PointerDataPacket& packet);
+  void DispatchKeyDataPacket(const KeyDataPacket& packet, uint64_t response_id);
```

Comment:
This should have a comment (as should all the others, but they are out of scope for this PR).